### ISSUE TITLE
downloadItem.cancel is not removing downloadItem from the queue

### ISSUE THREAD

**Issue by apphelpyulongs** [electron/electron#11130]
* Electron version:
* Operating system:
windows 8.1
### Expected behavior
downloadItem.cancel should remove the downloadItem from the queue

downloadItem.cancel should remove the downloadItem from the queue
### Actual behavior

even explicitly called downloadItem.cancel() the process is still ongoing
### How to reproduce
1. call webContents.downloadURL(this.downloadUrl);
2. call downloadItem.cancel() once the download has started
3. call webContents.downloadURL(this.downloadUrl) again
4. wait for it to finish if you put log in state completed you will see success twice
```
item.once('done', (event, state) => {
 if(state === 'completed') {
    console.log('success')
}
}

)
```

**Comment by sofianguy** [comment]
We are no longer providing bug fixes for 2.0.x or earlier. Is this bug still present in 2.0.x or later?

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
